yt_dlp/extractor/rai.py

```diff
@@ -735,3 +699,67 @@ def _real_extract(self, url):
         info.update(relinker_info)
 
         return info
+
+
+class RaiNewsIE(RaiIE):
+    _VALID_URL = r'https?://(www\.)?rainews\.it/.+?-(?P<id>%s)(?:-.+?)?\.html' % RaiBaseIE._UUID_RE
```

Comment:
```suggestion
    _VALID_URL = rf'https?://(www\.)?rainews\.it/[^?#]+-(?P<id>{RaiBaseIE._UUID_RE})(?:-[^/?#]+)?\.html'
```

Reply:
are we starting to use f-string?

Reply:
Yes. The minimum supported Python is 3.7 now